src/hello/store.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+
+import {observable, action} from 'mobx'
+
+class HelloStore {
+  @observable message: string = 'Hello.'
+
+  @action
```

Comment:
TBH, I'd prefer to have the `@action` decorator on the same line as the method.

Reply:
@ericnishio Can't we change the formatting to allow that?

Reply:
@crisu83 Yea. I just haven't found the right ESLint rule for it.

I don't mind having the `@observer` decorator above the `class` keyword. In fact, I prefer it that way. But `@action` (for methods), `@observable` (for properties), and `@computed` (for getters) should be placed on the same line.

It's only a matter of taste so some might disagree :)